lib/pages/DesktopScout.dart

```diff
@@ -56,6 +61,12 @@ class _DesktopScoutState extends State<DesktopScout> {
     double sportFilterIconSz = 14;
     double _width = MediaQuery.of(context).size.width;
     double _height = MediaQuery.of(context).size.height;
+    final mlbRepo = MLBRepo(MLBAthleteAPI(Dio()));
+    mlbRepo.getAllPlayers().then((response) =>
+        {print("First MLB player retrieved: ${response.first.name}")});
+    final nflRepo = NFLRepo(NFLAthleteAPI(Dio()));
+    nflRepo.getAllPlayers().then((response) =>
+        {print("First NFL player retrieved: ${response.first.name}")});
```

Comment:
This is out of scope for this ticket we can handle Refactoring the way we call NFL endpoints later